docs/product-model.md

```diff
@@ -619,6 +653,39 @@ An Index product is an exception because it's not directly tradable,
 but is included here because it can be referenced as an underlier for a
 tradable product and can be identified by a public identifier.
 
+### Observable
+
+The `Observable` data type specifies the reference object to be observed
+for a price, which could be an underlying asset or a reference to something
+whcih can be observed but not actually transfered, such as an index or
+a foreign exchange rate.
+
+The Observable data type requires the specification of either a
+`rateOption` (i.e. a floating rate index), `commodity`,
+`productIdentifier`, 
+`identifier` of an Asset,
+or `currencypair`. This choice constraint is
+supported by specifying a one-of condition, as shown below:
```

Comment:
the example Haskell below is a required choice and not a one-of?

Reply:
Neither the model or documentation has changed (the position of this text has moved to improve readability).  It is a `required choice` that excludes the `optionReferenceType` attribute.